cypress/e2e/immutableDatabaseTests/app-spec.ts

```diff
@@ -31,22 +31,6 @@ describe('Logged in Dockstore Home', () => {
     cy.scrollTo('bottom');
     cy.get('[data-cy=mt-toot]').should('be.visible');
   });
-
-  function starColumn(url: string, type: string) {
```

Comment:
Why was this removed?

Reply:
Oh, I see, it was never invoked. Never mind.

Reply:
It said it wasn't used and was catching the noUnusedLocals error